src/main/frontend/components/onboarding.cljs

```diff
@@ -23,25 +23,28 @@
                          [:span.mr-1 (t :help/shortcuts)]
                          (ui/icon "command" {:style {:font-size 20}})]]]
                       [(t :help/docs) "https://docs.logseq.com/"]
+                      [(t :help/start) "https://docs.logseq.com/#/page/tutorial?anchor=ls-block-62b17063-135d-4176-b755-fb5841556044"]
```

Comment:
This link has expired since block ids change on every build. Maybe just point to the root page for now?

Reply:
Good catch. Wanted to link to that specific block. But I guess the in it's own shouldn't be too big an issue. Fixed it.